### DISCUSSION TITLE
Why downgrade to AOMEnc v3.2.0?

### DISCUSSION THREAD

**Discussion by Kartus22** [staxrip/staxrip#985] [Q&A]
Hi,

Why do you guys downgrade to AOMenc 3.2 in the new Version? Is there any Reason for?

Even with Patmans, ETA Still not working in Vapoursynth with Filters on or after 1 - 2 Encodes.
Using ffms2
Piped Input still dont show ETA.

Greetings

**Comment by Dendraspis** [comment]
There was no downgrade, I just released it with an old(er) version of AOMEnc that I already added last year. A newer version is going to come...
